src/helpers/handlers/tests/handlers.test.tsx

```diff
@@ -16,7 +16,7 @@ describe('helpers/handlers', () => {
         y: 477.1875,
       },
       target: {
-        queryRenderedFeatures: (e: any) => {
+        queryRenderedFeatures: () => {
```

Comment:
What is the advantage of changing some of these unused params to `_` vs simply dropping them like this?

Reply:
I thought a little about this.  What I came up with is this, for unused function params:

- if it is the only one, just drop it
- if it precedes any other used param, replace it with `_`
- if it the last one, drop it

@cKellyDesign how can we make that better?

Reply:
I think that's perfect, I think I was just thrown the first instance where the only param was an array with 2 items. In that case we, even though it was unused we couldn't drop the second item because it wasn't a param.